test/integration/test_integration.py

```diff
@@ -77,7 +66,7 @@ def test_collection_lifecycle(collection_manager: CollectionManager):
     finally:
         # Delete collection
         if collection_manager.client.collections.exists("Movies"):
-            collection_manager.delete_collection(collection="Movies", all=False)
```

Comment:
Is this on purpose?

Reply:
Yes, because the default for all is already False, so it simply removes it. I could also remove the collection argument, but I think I added some clarity to what we were trying to delete